src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -510,7 +510,7 @@ kernelVersionTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property
 {
   //please append the new supported versions
   const std::vector<std::string> ubuntu_kernel_versions = { "4.4.0", "4.13.0", "4.15.0", "4.18.0", "5.0.0", "5.3.0" };
```

Comment:
Just curious about why are these versions hard-coded in the source?

Reply:
Until the new kernels are thoroughly tested, we don't support them officially.

Reply:
OK, but it does not prevent XRT to run, I hope?

Reply:
No, it doesn't. It simply gives a warning saying:
`Kernel verison %s is not officially supported. %s is the latest supported version`